skopt/tests/test_searchcv.py

```diff
@@ -77,6 +108,16 @@ def test_searchcv_runs(surrogate, n_jobs):
     assert_greater(opt.score(X_test, y_test), 0.9)
 
 
+def test_parallel_cv():
```

Comment:
This takes a few seconds to run. Please mark with `@pytest.mark.slow_test`.

Reply:
@iaroslav-ai Done!